templates/quickstart-jira-dc-with-vpc.template.yaml

```diff
@@ -333,39 +326,64 @@ Parameters:
     Default: ""
     Description: 'Use custom existing DNS name for your Data Center instance. This will take precedence over HostedZone. Please note: you must own the domain and configure it to point at the load balancer.'
     Type: String
+  # This parameter will be enabled when support for Aurora is added. Currently "dark" feature.
```

Comment:
Nit. We could make this a dropdown of 1 item, comment out the Aurora option and retain the dBEngine parameter, perhaps?

Reply:
I think that's probably confusing for end users. They might think why is there a drop down with only one option

Reply:
I agree, right now it is less confusing and adding/uncommenting Aurora related changes will be quick and we have a ticket to track it. I think we can merge this.